rewrite-yaml/src/test/java/org/openrewrite/yaml/ChangePropertyKeyTest.java

```diff
@@ -468,13 +471,16 @@ void relocatesPropertyWithSamePrefix() {
             """
               spring:
                 elasticsearch:
-                    restclient.sniffer.interval: 1
+                    restclient:
+                      sniffer:
+                        interval: 1
               """
           )
         );
     }
 
     @Nested
+    @Disabled
     @Issue("https://github.com/openrewrite/rewrite-spring/issues/189")
     class WhenOldPropertyKeyIsPrefixOfDotSeparatedKeyTest implements RewriteTest {
```

Comment:
What should we do about these test cases? Looked to be added for https://github.com/openrewrite/rewrite-spring/issues/189 by @nmck257 
Only when I add an explicit non null, non default separate does the first one pass again.

Does that mean there's a risk we break things for people that were using the recipe as it was before a merge?

Reply:
Thanks for the tag -- hadn't seen this PR.
Conceptually, I'm always going to be in favor of these yml utilities better-preserving existing style in the source code when it comes to dot separation vs indents. Will review these changes more specifically soon

Reply:
I believe `WhenOldPropertyKeyIsPrefixOfDotSeparatedKeyTest` is mostly here to contrast with the `ExceptTest` below it. So in that sense, the exact output of these particular cases is not super-important, because `rewrite-spring` is now doing what's shown in `ExceptTest` for this specific property migration, not what's shown in `WhenOldPropertyKeyIsPrefixOfDotSeparatedKeyTest`.